tests/read_frame.h

```diff
@@ -52,6 +52,35 @@ int read_frames(const std::string& filename) {
     return 1;
   }
 
+  // Reading specific (jumping to) entry
+  {
+    auto frame = podio::Frame(reader.readEntry("events", 4));
+    processEvent(frame, 4, reader.currentFileVersion());
+
+    auto otherFrame = podio::Frame(reader.readEntry("other_events", 4));
+    processEvent(otherFrame, 4 + 100, reader.currentFileVersion());
+  }
+
+  // Reading next entry after jump should continue the sequence
+  if (!reader.readNextEntry("events")) {
+    std::cerr << "Trying to read next entry after jump, should read event 5 out of 10" << std::endl;
+    return 1;
+  }
```

Comment:
```suggestion
  {
    auto nextFrame = podio::Frame(reader.readNextEntry("events"));
    processEvent(nextFrame, 5, reader.currentFileVersion());
  }
```
I was thinking more along the lines of actually checking whether this is really the correct next event here. Similar for the other cases.

Having said that, a check that reading past the available entries indeed returns a `nullptr` also seems like a good idea to have here.